src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2753,9 +2753,18 @@ def list_consumption_locations(cmd):
 
 
 def list_locations(cmd, sku, linux_workers_enabled=None):
-    client = web_client_factory(cmd.cli_ctx)
+    web_client = web_client_factory(cmd.cli_ctx)
     full_sku = get_sku_name(sku)
-    return client.list_geo_regions(full_sku, linux_workers_enabled)
+    web_client_geo_regions = web_client.list_geo_regions(full_sku, linux_workers_enabled)
```

Comment:
we will recommend to avoid positional arguments

Reply:
Fixed!